src/inventory/inventory-landing-page.jsx

```diff
@@ -39,13 +42,25 @@ export const InventoryLandingPage = () => {
 	const handleSearch = (event) => {
 		setSearchText(event.target.value);
 	};
+	const handleExportToExcel = () => {
+		exportToExcel(filteredRows);
+	  };
 
+	   
 	if (items?.results?.length > 0) {
 		for (let index = 0; index < items.results.length; index++) {
+			const item = items.results[index];
+			const expiration = item.details[0]?.expiration;
+			const expirationDate = new Date(expiration);
+			const formattedExpirationDate = `${expirationDate.getDate().toString().padStart(2, '0')}-${
```

Comment:
please move date calculation to a separate file, there is a file already called date-util.js. Please check
if you can reuse the existing function or move this logic to that file.